### DISCUSSION TITLE
This program seems not to track the switching of audio output devices.

### DISCUSSION THREAD

**Discussion by yiccx123** [rustdesk/rustdesk#7051] [General]
### Bug Description

For example, when I connect Bluetooth headphones, it will continue to use the previously set default output device to play the sound of remote nodes.

### How to Reproduce

Connect the Bluetooth headphones and set them as the system's default output device. The sound output for remote nodes still corresponds to the previous default audio output device.

Disconnecting Bluetooth headphones also reveals such an issue (the program does not automatically use the computer's built-in microphone to output the sound of remote nodes).

### Expected Behavior

It does not automatically switch to my Bluetooth headphones.

### Operating system(s) on local side and remote side

Windows 10 -> Windows 10

### RustDesk Version(s) on local side and remote side

1.2.3 -> 1.2.3

### Screenshots

This issue seems challenging to demonstrate through screenshots.

### Additional Context

No more, only the descriptions mentioned above.

**Comment by traycerb** [comment]
Yes, have the same issue.  Maybe it's challenging for the software to track the default audio output on the client when it changes mid-session, so perhaps there can be an option to choose the output for the sound.

**Comment by Zazulas** [comment]
So, two years later, this issue is still unresolved?
